docs/jade-getting-started.md

```diff
@@ -367,6 +367,8 @@ export GOOGLE_ALLOWREUSEEXISTINGPROJECTS=true
 export AZURE_CREDENTIALS_HOMETENANTID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
 export AZURE_CREDENTIALS_APPLICATIONID=<value from the top of .github/workflows/int-and-connected-test-run.yml>
 export AZURE_CREDENTIALS_SECRET=$(cat /tmp/jade-dev-azure.key)
+export AZURE_SYNAPSE_SQLADMINUSER=$(vault read -format=json secret/dsde/datarepo/dev/helm-azure | jq -r '.data["synapse-us-east-sql-admin-user"]')
```

Comment:
I think @fboulnois was trying to avoid reading from vault directly in this bit. I think the approach would be to move the logic here into `./render-configs.sh` then read the secret from /tmp/jade-dev-azure-synapse-sql-user.key, etc.

Reply:
Ah, gotcha - sounds good. Should be moved now!

Reply:
Thank you :)

Reply:
Reading everything in `~/.bash_profile` or equivalent dramatically slows down terminal startup.